java/ql/src/experimental/Security/CWE/CWE-275/plutoRce.ql

```diff
@@ -0,0 +1,25 @@
+import java
+import semmle.code.xml.WebXML
+
+class ValidHTTPMethods extends string {
+  ValidHTTPMethods() {
```

Comment:
This is named `ValidHttpMethods` as the other `getHttpMethods` calls returns the values preset in the databse. This returns hardcoded predefined strings. `HttpMethod` is quite similar to the `getHttpMethods` call. Hence, to prevent confusion, I named it the same.

Reply:
Interestingly, `PATCH`is not a default method, it is considered as an extension to the actual standard and is part of a eparate [RFC](https://tools.ietf.org/html/rfc5789). I have incldued it nevertheless. But I am concerned with the number of false positives this may cause.